account_invoice_commission/README.rst

```diff
@@ -48,23 +56,22 @@ To configure this module, you need to:
 #. No configuration nedeed.
 
 
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.adhoc.com.ar/
+Usage
+=====
 
-.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
-.. branch is "8.0" for example
+To use this module, you need to:
 
-Known issues / Roadmap
-======================
+#. Go to ...
 
-* ...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: http://runbot.adhoc.com.ar/
```

Comment:
```
+   :target: http://runbot.adhoc.com.ar/
```
Need to be https: http page does not work.

Reply:
sorry, my mistake. it's the opposite. http is the one that works.
Do not take into account this comment.